### PR TITLE
 All Task Completed

### PR DESCRIPTION
**Pull request by zhafranafif**
- [ ] Completed Praktikum Task
- [ ] Completed Screenshot
- [ ] Completed Summary Task

### CODE REVIEW

**Comment by syukronarie** [comment]
> * [ ]  Completed Praktikum Task
> * [ ]  Completed Screenshot
> * [ ]  Completed Summary Task

bro buat description kedepannya harus lebih informative

File: 06_CSS/praktikum/file1.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        }
+        #box-2{
+            background-color: blue;
+            padding: 30px;
+        }
+        #box-3{
+            background-color: green;
+            padding: 0 10px 100% 10px;
+            border-radius: 12px;
+        }
```
Comment (syukronarie) [inline 06_CSS/praktikum/file1.html:41]: klo standard styling background, padding , border, baiknya pake class name saja bro

**Review by syukronarie — APPROVED** [review]
Reviewed. Tks

File: 06_CSS/praktikum/file1.html (hunk 1 above)
Comment (zhafranafif) [reply 06_CSS/praktikum/file1.html:41]: baik mas ariee, terima kasih

**Pull request merged**